scripts/firedrake-install

```diff
@@ -1952,13 +1921,20 @@ if mode == "install":
                 install(p+"/")
             sys.path.append(os.getcwd() + "/" + p)
 
+    with environment(**compiler_env):
+        build_and_install_libsupermesh()
+
+    with pipargs("--no-deps"):
```

Comment:
Put the context managers together? https://stackoverflow.com/questions/893333/multiple-variables-in-a-with-statement